src/tasks/publish/subtasks/getUpdateFilesTask.ts

```diff
@@ -65,19 +63,17 @@ export function updateVariantFiles({
   variantsDirPath,
   dnpName,
   nextVersion,
-  isMultiVariant
 }: {
   rootDir: string;
   composeFileName: string;
-  variant: string;
+  variant: string | null;
```

Comment:
why allowing null? ist better the usage of a boolean?

Reply:
If I added a boolean, `variant` would still have to be `undefined` or `null` for single packages, so I would say it does not provide any value.